src/components/Film/index.js

```diff
@@ -1,56 +1,57 @@
-import React,{ useState, useEffect } from "react";
-import './Film.css';
-import { FaArrowRight } from 'react-icons/fa';
-import { FilmWatched } from '../FilmWatched';
-import axios from "axios";
+import React, { useContext } from "react";
```

Comment:
:v the thing was that I found an error in the Ivan's PR, so I asked you "what changes do I take xd?....the incoming?" and you said  "Yes!, (I took that answer xD)" "the useContext one (and it wasn't the incoming, it was the "Head") "(which I didn't understood but I thought "well, Lili said "yes" to the incoming changes") and I did those changes... sent the profile changes (with the useContext deleted because of the misunderstanding xD) that's why Ivan said my PR had a "problem" carried from his PR xD,  so I "fixed" that just making it come back to your changes xD, that's why this are the same changes you "have" xD

Reply:
Got it. Sorry for that miscommunication. I didn't realice that Ivan hadn't merged the remote changes, as he created the change of context, that's why I texted you the useContext one.